did_peer/src/error.rs

```diff
@@ -0,0 +1,69 @@
+use std::convert::Infallible;
+
+use did_doc::schema::verification_method::VerificationMethodType;
+use thiserror::Error;
```

Comment:
This is a very very very small thing, but IMO it's a good practice to alias stuff that are common in the `std`. Usually `Error` refers to the `std::error::Error` trait. Disregard if it's unimportant to you.

Reply:
No, I agree. Just missed this, thanks for pointing it out.

Reply:
Actually, I realized you are not talking about an error type, but the thiserror derive macro implementing `std::error::Error` trait. I think this is fine, it's clear from the context that it is a macro, not a trait, so no confusion with `std::error::Error` should occur, and `thiserror` documentation imports the macro in the same way.